src/networks/devnet/mod.rs

```diff
@@ -159,17 +155,11 @@ fn get_bundle_cid(version: &str) -> Cid {
         .bundle_cid
 }
 
-pub(super) static DRAND_SCHEDULE: Lazy<[DrandPoint<'static>; 2]> = Lazy::new(|| {
-    [
-        DrandPoint {
-            height: 0,
-            config: &DRAND_MAINNET,
-        },
-        DrandPoint {
-            height: get_upgrade_height_from_env("FOREST_DRAND_QUICKNET_HEIGHT").unwrap_or(i64::MAX),
-            config: &DRAND_QUICKNET,
-        },
-    ]
+pub(super) static DRAND_SCHEDULE: Lazy<[DrandPoint<'static>; 1]> = Lazy::new(|| {
+    [DrandPoint {
+        height: 0,
+        config: &DRAND_QUICKNET,
```

Comment:
Just to clarify: `DRAND_MAINNET` has been removed, because it's not needed in DEVNET setup, right?

Reply:
See the PR description.